src/index.js

```diff
@@ -1,10 +1,11 @@
 import React, { useState } from "react";
 import ReactDOM from "react-dom";
 import "./styles.css";
-import WebTorrent, { ClientEvents } from "./webtorrent-custom";
+import WebTorrent from "./webtorrent-custom";
```

Comment:
Can we rename this to `web3torrent-lib` or something like that?

Reply:
Sure! I'll make it in another PR, so we don't lose traceability of the changes done in this PR